lib/engine/config/game/g_18_co.rb

```diff
@@ -0,0 +1,1260 @@
+		"co6": {
+			"count": 1,
+			"color": "green",
+			"code": "city=revenue:40,slots:2;path=a:0,b:_0;path=a:2,b:_0;path=a:3,b:_0;path=a:4,b:_0;path=a:5,b:_0;label=CS;"
+		},
+		"39": 1,
+		"40": 2,
+		"41": 1,
+		"42": 1,
+		"43": 1,
+		"44": 1,
+		"45": 1,
+		"46": 1,
+		"47": 1,
+		"63": 6,
+		"co3": {
+			"count": 1,
+			"color": "brown",
+			"code": "city=revenue:70,slots:4;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;path=a:4,b:_0;path=a:5,b:_0;label=D;"
+		},
+		"co4": {
+			"count": 3,
+			"color": "brown",
+			"code": "city=revenue:50,slots:2;path=a:0,b:_0;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;path=a:4,b:_0;"
+		},
+		"co7": {
+			"count": 1,
+			"color": "brown",
+			"code": "city=revenue:60,slots:3;path=a:0,b:_0,;path=a:1,b:_0;path=a:2,b:_0;path=a:3,b:_0;path=a:4,b:_0;path=a:5,b:_0;label=CS;"
+		}
+	},
+	"market": [
+		[
+			"140",
+			"145",
+			"150",
+			"155",
+			"165",
+			"175",
+			"190",
+			"205",
+			"225",
+			"250",
+			"280",
+			"315",
+			"355",
+			"395",
+			"440",
+			"485"
+		],
+		[
+			"110",
+			"115",
+			"120",
+			"125",
+			"135",
+			"145p",
+			"160p",
+			"175",
+			"195",
+			"220",
+			"250",
+			"280",
+			"315",
+			"350",
+			"385",
+			"425"
+		],
+		[
+			"85",
+			"90",
+			"95",
+			"100p",
+			"110p",
+			"120p",
+			"135p",
+			"150",
+			"170",
+			"195",
+			"220",
+			"245",
+			"275",
+			"305",
+			"335",
+			"370"
+		],
+		[
+			"65",
+			"70",
+			"75p",
+			"80p",
+			"90p",
+			"100",
+			"115",
+			"130",
+			"150",
+			"170",
+			"195",
+			"215",
+			"240",
+			"265",
+			"290",
+			"320"
+		],
+		[
+			"50",
+			"55",
+			"60p",
+			"65",
+			"75",
+			"85",
+			"100",
+			"115",
+			"130",
+			"150",
+			"170",
+			"185",
+			"205"
+		],
+		[
+			"40",
+			"45",
+			"50p",
+			"55",
+			"65",
+			"75",
+			"85",
+			"100",
+			"115",
+			"130"
+		],
+		[
+			"30",
+			"35",
+			"40p",
+			"45",
+			"55",
+			"65",
+			"75",
+			"85"
+		],
+		[
+			"25",
+			"30",
+			"35",
+			"40",
+			"45",
+			"55",
+			"65"
+		],
+		[
+			"20",
+			"25",
+			"30",
+			"35",
+			"40",
+			"45"
+		],
+		[
+			"15",
+			"20",
+			"25",
+			"30",
+			"35",
+			"40"
+		],
+		[
+			"10b",
+			"15",
+			"20",
+			"25",
+			"30"
+		],
+		[
+			"10b",
+			"10b",
+			"15",
+			"20",
+			"20"
+		],
+		[
+			"10b",
+			"10b",
+			"10b",
+			"15b",
+			"15b"
+		]
+	],
+	"companies": [
+		{
+			"sym": "IMC",
+			"name": "Idarado Mining Company",
+			"value": 30,
+			"revenue": 5,
+			"desc": "Money gained from mine tokens is doubled for the owning Corporation. If owned by a Corporation, closes on purchase of “6” train, otherwise closes on purchase of “5” train.",
+			"abilities": [
+
+			]
+		},
+		{
+			"sym": "GJGR",
+			"name": "Grand Junction and Grand River Valley Railway",
+			"value": 40,
+			"revenue": 10,
+			"desc": "An owning Corporation may upgrade a yellow town to a green city in additional to its normal tile lay. Action closes the company or closes on purchase of “5” train.",
```

Comment:
Check if there is a way to express via an ability? As it is now, the use will just remove the ability. But doing it in code is OK.

Reply:
This is one of those pieces I was in the middle of working when Jen suggested I get a PR up before I get too deep. I haven't actually had a chance to test this bit yet. Should I just remove the ability completely and leave it for a later PR?

Side question: Why would it just remove the ability? I read it as "Lay a tile of 14 or 15 in one of the following hexes" ... I know that isn't quite accurate to the ability, but it seemed it should have done something close to what I wanted.